azure-functions-java-spi/pom.xml

```diff
@@ -0,0 +1,113 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.azure.functions</groupId>
+    <artifactId>azure-functions-java-spi</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+    <parent>
+        <groupId>com.microsoft.maven</groupId>
+        <artifactId>java-8-parent</artifactId>
+        <version>8.0.1</version>
+        <relativePath></relativePath>
+    </parent>
+
+    <name>Microsoft Azure Functions Java SPI Types</name>
+    <description>This package contains all SPI interfaces for third parties to interact with Microsoft Azure functions runtime.</description>
+    <url>https://azure.microsoft.com/en-us/services/functions</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://opensource.org/licenses/MIT</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:https://github.com/Azure/azure-functions-java-core-library</connection>
+        <developerConnection>scm:git:git@github.com:Azure/azure-functions-java-core-library</developerConnection>
+        <url>https://github.com/Azure/azure-functions-java-core-library</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <developers>
+        <developer>
+            <id>Microsoft</id>
+            <name>Microsoft Corporation</name>
+        </developer>
+    </developers>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <name>Sonatype Snapshots</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+            <uniqueVersion>true</uniqueVersion>
+            <layout>default</layout>
+        </snapshotRepository>
+    </distributionManagement>
+
```

Comment:
see if this can be removed, looks like it's inherited from parent https://search.maven.org/artifact/com.microsoft.maven/java-parent-bom/1.0.2/pom